engine/src/execution_graph/logic_controllers/CacheMachine.cpp

```diff
@@ -474,6 +474,17 @@ std::unique_ptr<ral::frame::BlazingTable> ConcatenatingCacheMachine::pullFromCac
 			tables_holder[i] = std::move(data->decache());
 			table_views[i] = tables_holder[i]->toBlazingTableView();
 		}
+
+		if( concat_all && ral::utilities::checkIfConcatenatingStringsWillOverflow(table_views) ) {
```

Comment:
Here we are throwing a warning if concat_all is true and if concatenating strings will overflow. That is ok, to just throw a warning because we cant do anything about it. But if concat_all is false, and concatenating strings will overflow, then we do need to do something about it